data/pathfinder_pfs/paizo/campaign_setting/pfs_dark_markets/pfs_dm_feats.lst

```diff
@@ -2,7 +2,7 @@
 SOURCELONG:Dark Markets - A Guide to Katapesh	SOURCESHORT:DM	SOURCEWEB:http://paizo.com/pathfinder/campaignSetting/35E/v5748btpy87ut	SOURCEDATE:2009-04
 # Original Entry by: Terry Milnes
 
-#Pesh Euphoria.MOD		VISIBLE:NO	TYPE:PFSNotLegal
-#Pesh Healing.MOD		VISIBLE:NO	TYPE:PFSNotLegal
-#Pesh Rejuvenation.MOD		VISIBLE:NO	TYPE:PFSNotLegal
-#Sahir-Afiyun.MOD		VISIBLE:NO	TYPE:PFSNotLegal
+#CATEGORY=FEAT|Pesh Euphoria.MOD		!PRECHARACTERTYPE:1,PC	TYPE:PFSNotLegal
+#CATEGORY=FEAT|Pesh Healing.MOD		!PRECHARACTERTYPE:1,PC	TYPE:PFSNotLegal
+#CATEGORY=FEAT|Pesh Rejuvenation.MOD		!PRECHARACTERTYPE:1,PC	TYPE:PFSNotLegal
+#CATEGORY=FEAT|Sahir-Afiyun.MOD		!PRECHARACTERTYPE:1,PC	TYPE:PFSNotLegal
```

Comment:
These are items that are listed in multiple sources. PFS rules refer to them in this source specifically, so they are listed.

 

From: Andrew Maitland [mailto:notifications@github.com] 
Sent: Monday, January 26, 2015 12:33 AM
To: PCGen/pcgen
Cc: mjmeans
Subject: Re: [pcgen] DATA-2327 - PFS updates (#132)

 

In data/pathfinder_pfs/paizo/campaign_setting/pfs_dark_markets/pfs_dm_feats.lst <https://github.com/PCGen/pcgen/pull/132#discussion_r23515172> :

> @@ -2,7 +2,7 @@
>  SOURCELONG:Dark Markets - A Guide to Katapesh     SOURCESHORT:DM        SOURCEWEB:http://paizo.com/pathfinder/campaignSetting/35E/v5748btpy87ut        SOURCEDATE:2009-04
>  # Original Entry by: Terry Milnes
>  
> -#Pesh Euphoria.MOD          VISIBLE:NO     TYPE:PFSNotLegal
> -#Pesh Healing.MOD           VISIBLE:NO     TYPE:PFSNotLegal
> -#Pesh Rejuvenation.MOD             VISIBLE:NO     TYPE:PFSNotLegal
> -#Sahir-Afiyun.MOD           VISIBLE:NO     TYPE:PFSNotLegal
> +#CATEGORY=FEAT|Pesh Euphoria.MOD           !PRECHARACTERTYPE:1,PC TYPE:PFSNotLegal
> +#CATEGORY=FEAT|Pesh Healing.MOD            !PRECHARACTERTYPE:1,PC TYPE:PFSNotLegal
> +#CATEGORY=FEAT|Pesh Rejuvenation.MOD              !PRECHARACTERTYPE:1,PC        TYPE:PFSNotLegal
> +#CATEGORY=FEAT|Sahir-Afiyun.MOD            !PRECHARACTERTYPE:1,PC TYPE:PFSNotLegal

If these are commented out why are we even listing them?
Why not decomment them?

—
Reply to this email directly or view it on GitHub <https://github.com/PCGen/pcgen/pull/132/files#r23515172> .  <https://github.com/notifications/beacon/ADyap3Ylhd4rYZo_YE2e-bLcyHXsdPfiks5nleUagaJpZM4DXDYk.gif>